dot_lpdev_cmds.sh

```diff
@@ -251,7 +251,7 @@ function __lpdev_geth_run {
        -rpccorsdomain "*" -wsorigins "*" \\
        -rpcapi 'personal,account,eth,web3,net' \\
        -wsapi 'personal,account,eth,web3,net' \\
-       -targetgaslimit 6700000 \\
+       -targetgaslimit 8000000 \\
```

Comment:
I'm curious - why are we increasing the gas limit?

Reply:
Contract deployment on the [pm](https://github.com/livepeer/protocol/blob/pm/truffle.js#L50) branch uses a 8m gas limit so this adjustment is just to make sure that block gas limit is also as high. There is no strong reason for setting this hardcoded gas limit besides making sure that no matter how big contracts get during development there are few cases where the gas limit attached to a transaction is too little. This is actually not ideal for production deployment because we would want to estimate the gas required for each transaction instead of setting a really high gas limit for each transaction, but this change is ok for the development environment.

Reply:
Thanks for the explanation @yondonfu !